pkgs/applications/window-managers/eww/default.nix

```diff
@@ -5,32 +5,26 @@
 , wrapGAppsHook
 , gtk3
 , librsvg
-, withWayland ? false
 , gtk-layer-shell
 , stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "eww";
-  version = "unstable-2023-08-18";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "elkowar";
     repo = "eww";
-    rev = "a9a35c1804d72ef92e04ee71555bd9e5a08fa17e";
-    hash = "sha256-GEysmNDm+olt1WXHzRwb4ZLifkXmeP5+APAN3b81/Og=";
+    rev = "387d344690903949121040f8a892f946e323c472";
+    hash = "sha256-06d8pm1cyg0zpa1kcnzzja2ia3c4glyr6ngkw1hs6l1pivj64kza";
   };
 
   cargoHash = "sha256-4yeu5AgleZMOLKNynGMd0XuyZxyyZ+RmzNtuJiNPN8g=";
```

Comment:
You might update `cargoHash` if any of cargo dependencies were updated.

Reply:
use `lib.fakeHash` as it is written in nixpkgs manual about Rust packaging

Reply:
Yes, it needs to be updated:

> error: build of '/nix/store/qqr5gpd4fjr1pvm3m7zqgqjbck1bz6gl-eww-0.5.0-vendor.tar.gz.drv' on - failed: hash mismatch in fixed-output derivation '/nix/store/qqr5gpd4fjr1pvm3m7zqgqjbck1bz6gl-eww-0.5.0-vendor.tar.gz.drv':
         specified: sha256-4yeu5AgleZMOLKNynGMd0XuyZxyyZ+RmzNtuJiNPN8g=
            got:    sha256-IirFE714NZmppLjwbWk6fxcmRXCUFzB4oxOxBvmYu5U=